pyvista/plotting/__init__.py

```diff
@@ -11,6 +11,29 @@
 
 from .renderer import CameraPosition, Renderer, scale_point
 from .plotting import BasePlotter, Plotter, close_all
-from .qt_plotting import BackgroundPlotter, QtInteractor, MainWindow, Counter
 from .helpers import plot, plot_arrows, plot_compare_four, plot_itk
 from .itkplotter import PlotterITK
+
+
+
+class DeprecationError(Exception):
+    """Depreciation Error."""
+
+    pass
```

Comment:
Can we add a more informative message by default and call this `QtDeprecationError`

```py
class QtDeprecationError(Exception):
    """Depreciation Error for features that moved to `pyvistaqt`."""
    
    message = """`{}` has moved to pyvistaqt.
    You can install this from PyPI with: `pip install pyvistaqt`
    See https://github.com/pyvista/pyvistaqt
"""
    def __init__(self, feature_name):
        Exception.__init__(self, self.message.format(feature_name))
```

Reply:
That's more informative.  I'll patch that.